test/test_verify.py

```diff
@@ -564,10 +569,10 @@ def execution(testSet, testSuitList, testCleanEnableList, testSuitEnableList, dr
 
 
 if __name__ == "__main__":
-    if len(sys.argv) != 9:
+    if len(sys.argv) < 10:
```

Comment:
Since `skipProxy` and `allowedTestsCsv` are optional won't it fail when I skip them (I'll have 9 elements in argv).

Reply:
skipProxy is set to default value "false" so it is not optional from the `test_verify.py` perspective. But you could be mislead by comment so I marked `skipProxy` as required here.